app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+def copy_file(command: str) -> None:
+    # Split the command string into its components
+    parts = command.split()
```

Comment:
It can be done simpler

```suggestion
    cmd, source_file, dest_file = command.split()
```